src/libman/LibraryResolver.cs

```diff
@@ -21,16 +21,12 @@ internal static class LibraryResolver
         /// </summary>
         /// <param name="partialName">Can be display name or library id.</param>
         /// <param name="manifest"></param>
-        /// <param name="manifestDependencies"></param>
         /// <param name="provider"></param>
-        /// <param name="cancellationToken"></param>
         /// <returns></returns>
-        public static async Task<IReadOnlyList<ILibraryInstallationState>> ResolveAsync(
+        public static IReadOnlyList<ILibraryInstallationState> Resolve(
```

Comment:
Can we use the SearchAsync on the Catalogs for that?

Reply:
No, we don't need to, now that library name and version are already split. 
And also `SearchAsync` makes network calls.